### DISCUSSION TITLE
Ideas for Auto Deletion 🤷‍♂️

### DISCUSSION THREAD

**Discussion by Minxster** [sct/overseerr#3640] [General]
Just some thoughts & ideas, since I'm struggling to see how I can auto-remove watched items. And I know there are a million and one edge cases on why some of these are a bad idea or won't work. I'm just trying to drum up some discussion on how we (all) can efficiently look at removing media that's no longer needed by remote users.

1. Since Overseerr is reading (potentially) data from Tautulli, could we have a report to say that a requested show/film has now been watched by the person who requested it? At least that would give me a simple way to view what I can be delete?
2. Have the option, if Tautulli is in use, to auto-remove a film if it's been watched, directly from within Radarr
3. Basically the same as "2." but for Sonarr? Though I fully get this is a bit more problematic with episodes and seasons and needing to know what user is where in the show's seasons
4. Thinking on "Import Lists", could we have a Radarr and a Sonarr "list" per *arr and per user? Or even just one large list for each *arr? This could be connected into *arr instances so we could use the "Clean Library Level" option? I run 2nd instances of the *arrs so that I can use the clean-up library facility withing causing me issues with my core "always keep" media. 

I open to more ideas, and I know some of the above would be (a) a bad idea or (b) PITA!

**Comment by marchershey** [comment]
I vote for the ability to auto delete after a set number of days/months/etc.

As of now, I have a script that runs once a month and it will delete any media that is 4 months old. It uses the Arr's API to delete the media. It also moves my "purge day is coming up" pre-roll video into the pre-roll folder so my users are reminded that a purge is about to happen. 

I would love this to be centralized in Overseerr, but until this, it will just stay on my janky Laravel-made Media manager app that I made a while back.

**Comment by adrianguanipa** [comment]
Did you try Maintainerr? that also supports overseerr

https://github.com/jorenn92/Maintainerr

**Reply by marchershey** [reply]
Thanks for this comment. 

I recently discovered Maintainerr and boy oh boy, its awesome!
